draft-barnes-acme.md

```diff
@@ -344,12 +347,16 @@ provided by the client.
 The server creates a registration object with the included contact information.  The "key" element of the registration is set to the public key used to verify the JWS (i.e., the "jwk" element of the JWS header).  The server also provides a random
 recovery token.  The server returns this registration object in a 201 (Created) response, with the registration URI in a Location header field.  The server may also indicate its new-authorization URI using the "next" link relation.
 
+If the server wishes to present the client with terms under which the ACME service is to be used, it may indicate the URI where such terms can be accessed in a Link header with link relation "terms-of-service".  As noted above, the client may indicate its
```

Comment:
I guess the point is that the client might reference an old, but perfectly acceptable version.  The server has to decide whether it needs to reject the request.

Is there any automated way to signal that the agreement was not current or correct?